src/status_im/mailserver/core.cljs

```diff
@@ -332,24 +332,29 @@
 
 (fx/defn process-next-messages-request
   [{:keys [db now] :as cofx}]
-  (when (and (transport.db/all-filters-added? cofx)
-             (not (:mailserver/current-request db)))
-    (when-let [mailserver (get-mailserver-when-ready cofx)]
-      (let [request-to (or (:mailserver/request-to db)
-                           (quot now 1000))
-            requests (prepare-messages-requests cofx request-to)
-            web3 (:web3 db)]
-        (if-let [request (first requests)]
-          {:db (assoc db
-                      :mailserver/pending-requests (count requests)
-                      :mailserver/current-request request
-                      :mailserver/request-to request-to)
-           :mailserver/request-messages {:web3     web3
-                                         :mailserver    mailserver
-                                         :request request}}
-          {:db (dissoc db
-                       :mailserver/pending-requests
-                       :mailserver/request-to)})))))
+  (let [network (:network/type db)
+        {:keys [syncing-on-mobile-network?]} (:account/account db)]
+    (when (and
+           (not (and (= network "cellular")
```

Comment:
You've defined a function called `cellular?` Maybe we can re-use it  instead of keeping the hardcoded values?

Reply:
fixed